src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -185,3 +186,232 @@ If you publish your template as *Unlisted*, you can test it under exactly the sa
+The *Info and Features pane* includes the following panels.
+
+==== Environment info
+
+The *Environment info* panel displays your templates's *App Id*, *Frame URL*, and *VCC editor source*.
+You can click *Modify* to edit the frame URL.
+
+The *Environment info* panel also allows you to edit the *VCC editor source*.
+When you publish a template, the VCC editor data defines the VCCs available in the template.
+So when the template calls something like `InstantRemixing.onPresentControl(['general', 'photo'])`, the editor looks in the data for the `general.photo` path and knows to show the image VCC.
+If you're working on a new version of a template that uses different VCCs from the currently published template, and you're using the debugger to override the URL and debug something running locally or running in the web IDE, you can edit the *VCC editor source* to override the live data.
+
+You edit the *VCC editor source* by clicking *Modify* and selecting *Override and use different VCC editor data*.
+This displays the data in JSON format.
+You can directly edit the JSON code to override the live values.
+
+For example, suppose the VCC editor data for a template reads as follows:
+
+[source, json]
+----
+[
+  {
+    "key": "general",
+    "name": "General",
+    "fields": [
+      {
+        "key": "name",
+        "name": "Name",
+        "type": "text"
+      },
+      {
+        "key": "title",
+        "name": "Title",
+        "type": "text"
+      },
+      {
+        "key": "description",
+        "name": "Description",
+        "type": "text"
+      },
+      {
+        "key": "profileSrc",
+        "name": "Profile Image",
+        "type": "image"
+      },
+      {
+        "key": "newMessageBackground",
+        "name": "New Message Background",
+        "type": "image"
+      },
+      {
+        "key": "photo",
+        "name": "Photo",
+        "type": "image"
+      },
+      {
+        "key": "video",
+        "name": "Video",
+        "type": "video"
+      },
+      {
+        "key": "price",
+        "name": "Price Per Message",
+        "type": "text"
+      }
+    ],
+    "source": "general.json"
+  }
+]
+----
+
+The following portion of the JSON data tells Koji that the template will use a profile image and, when remixing, the prompt for that image will be "Profile Image":
+
+[source, json]
+----
+{
+    "key": "profileSrc",
+    "name": "Profile Image",
+    "type": "image"
+},
+----
+
+Suppose you're working on a new version of the template that uses a profile video instead of a profile image.
+You can override the live data by changing `"type": "image"` to `"type": "video"`, and by changing `"name": "Profile Image",` to `"name": "Profile Video",`.
+Now, when working inside the debugger, the remixer will show a video VCC instead of an image VCC, and the prompt for changing the video will be "Profile Video".
+Without using the override, you would need to update the VCC definition and republish in order to preview the change.
```

Comment:
Needs something simpler and more straightforward to explain why this is important. Here's one suggestion. Also, consider making this point before you describe the example. It might flow better.

```suggestion
Overriding the VCC data enables you to test the new VCC definition without republishing the template.
```